### PR TITLE
change trigger

### PR DESCRIPTION
**Pull request by justinL23**

### CODE REVIEW

**Comment by justinL23** [comment]
@earloc ;)

**Comment by earloc** [comment]
hey thanks @justinL23 . But what does this actually change, especially regarding to #1 ?
Or is there any other pain that gets adressed with this?

File: .github/workflows/pullrequests.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 name: postomate
 
-on:
-  pull_request:
-    branches: [ main ]
+on: workflow_dispatch
```
Comment (earloc) [inline .github/workflows/pullrequests.yml:3]: hmm, ok, if we want the action to be triggerd on-demand. But couldn´t this just be put into main.yml, either?

Comment (earloc) [reply .github/workflows/pullrequests.yml:3]: guess this filoe here could just be deleted then?

**Review by earloc — CHANGES_REQUESTED** [review]

File: .github/workflows/pullrequests.yml (hunk 1 above)
Comment (v-braun) [reply .github/workflows/pullrequests.yml:3]: one day ... long long ago ... someone ... a wise man ... told me:
> why the heck you want to trigger a build on-demand? It should always be triggered by a code change ;-)

Comment (earloc) [reply .github/workflows/pullrequests.yml:3]: > If building you want, changing you must

-- Yoda, CI/CD-jedi-master

**Comment by earloc** [comment]
ok, again thx for the contrib. I´ll soon push another change containing these here as well.

File: .github/workflows/pullrequests.yml (hunk 1 above)
Comment (earloc) [reply .github/workflows/pullrequests.yml:3]: see #8

**Comment by earloc** [comment]
see #8

**Pull request closed without merging**